src/main/resources/db/migration/V1_5__adding_on_delete_cascade_tables.sql

```diff
@@ -0,0 +1,5 @@
+
+ALTER TABLE response DROP CONSTRAINT user_profile_id_fk1;
+alter table response add constraint user_profile_id_fk1 foreign key (user_profile_id)references user_profile (id) on delete cascade;
```

Comment:
response table should not have delete cascade as you will never be deleting it from response table

Reply:
currently we are deleting entries from both user_profile and response table and delete cascade adding on the user_profile_id_fk1 constraint

Reply:
https://tools.hmcts.net/jira/browse/RDCC-1805 i have created tech debt ticket for remove the foreign key constraint and modify the code